flavors/python-3.8-conda-minimal-EXASOL-6.2.0/flavor_base/conda_deps/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV ARCHIVE_UBUNTU_PREFIX=""
+RUN sed --in-place --regexp-extended "s/(\/\/)(archive\.ubuntu)/\1$ARCHIVE_UBUNTU_PREFIX\2/" /etc/apt/sources.list
+
+COPY 01_nodoc /etc/dpkg/dpkg.cfg.d/01_nodoc
+
+COPY scripts /scripts
+
+RUN mkdir -p /build_info/packages
+COPY conda_deps/packages /build_info/packages/conda_deps
+RUN /scripts/install_scripts/install_via_apt.pl --file /build_info/packages/conda_deps/apt_get_packages --with-versions
+
+ENV ENV_NAME="base"
+ENV MAMBA_ROOT_PREFIX="/opt/conda"
+ENV MAMBA_EXE="/bin/micromamba"
+ENV MAMBA_DOCKERFILE_ACTIVATE=1
+
+COPY conda_deps/_activate_current_env.sh /usr/local/bin/_activate_current_env.sh
+
+RUN addgroup --gid 1000 exasolution
+RUN adduser --disabled-login --uid 1000 --gid 1000 exasolution --gecos "First Last,RoomNumber,WorkPhone,HomePhone"
+RUN addgroup --gid 500 exausers 
+RUN adduser --disabled-login --uid 500 --gid 500 exadefusr --gecos "First Last,RoomNumber,WorkPhone,HomePhone"
+
+SHELL ["/bin/bash", "-l", "-c"]
+RUN /scripts/install_scripts/install_micromamba.sh 0.23.0
```

Comment:
Here you copy the _activate_current_env.sh from conda_deps, and then later you overwrite it when executing `install_micromamba.sh`. Is this intended?

Reply:
Yes, I embed the environment variables necessary to activate the conda environment